Win11ThemeTest/ButtonTest.cs

```diff
@@ -0,0 +1,160 @@
+﻿using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
+using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Conditions;
+using FlaUI.Core.Definitions;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
+using FlaUI.UIA3.Identifiers;
+using FlaUI.UIA3.Patterns;
+using System.Drawing;
+using System.Linq;
+using NuGet.Frameworks;
+
+
+namespace Win11ThemeTest
+{
+    public class ButtonTest
+    {
+        private Application app;
+        private Window window;
+        //  private Window window2;
+        public Window btnwindow;
+        Button testButton;
+        Button button;
+        Button disabledbutton;
+        // Button iconbutton;
+
+        public ButtonTest()
+        {
+
+            app = Application.Launch(@"..\\..\\..\\..\\TestingApplication\\bin\\Debug\\net9.0-windows\\win-x64\\TestingApplication.exe");
+
+            using (var automation = new UIA3Automation())
+            {
+                window = app.GetMainWindow(automation);
+                testButton = window.FindFirstDescendant(cf => cf.ByAutomationId("testbtn")).AsButton();
+                Mouse.Click(testButton.GetClickablePoint());
+                Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(2000));
+                btnwindow = window.FindFirstDescendant(cf => cf.ByName("ButtonWindow")).AsWindow();
+
+            }
+
+        }
+
+        //test if button is available in window
+        [Test]
+        public void btn1_isButtonAvailable()
+        {
+            Assert.IsNotNull(btnwindow);
+            button = btnwindow.FindFirstDescendant(cf => cf.ByAutomationId("btn")).AsButton();
```

Comment:
Since `button` and `disabledbutton` are non nullable so initialize them in ButtonTest() constructor. Move following initialization in the constructor -
```
disabledbutton = btnwindow.FindFirstDescendant(cf => cf.ByAutomationId("disbtn")).AsButton();
button = btnwindow.FindFirstDescendant(cf => cf.ByAutomationId("btn")).AsButton();
```

When the present implementation if you will run individual tests then you will get `NullReferenceException`.

Reply:
Initialization is added in constructor.